### PR TITLE
Fix pytest fixtures on Win10

### PR DESCRIPTION
**Pull request by Caligatio**
# Pull Request Check List

This is just a reminder about the most common mistakes. Please make sure that you tick all *appropriate* boxes.  But please read our [contribution guide](https://poetry.eustace.io/docs/contributing/) at least once, it will save you unnecessary review cycles!

- [ ] Added **tests** for changed code.
- [ ] Updated **documentation** for changed code.

**Note**: If your Pull Request introduces a new feature or changes the current behavior, it should be based
on the `develop` branch. If it's a bug fix or only a documentation update, it should be based on the `master` branch.

If you have *any* questions to *any* of the points above, just **submit and ask**!  This checklist is here to *help* you, not to deter you from contributing!

---
Two conftest.py files are currently attempting to perform symlinking as part of a fixture which requires either [admin privileges or "Developer Mode" to be enabled](https://blogs.windows.com/windowsdeveloper/2016/12/02/symlinks-windows-10/) on Win10.  This PR just wraps the symlink call in a try/except and falls back to copying if it fails.

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -87,8 +87,14 @@ def mock_download(self, url, dest):
     if dest.exists():
         os.unlink(str(dest))
 
-    if PY2 and WINDOWS:
-        shutil.copyfile(str(fixture), str(dest))
+    if WINDOWS:
```
Comment (brycedrennan) [inline tests/conftest.py]: I'm tempted to replace this whole block with:
```python
	    try:
                os.symlink(str(fixture), str(dest))
            except OSError:
                # symlink doesn't work on windows in python 2 or without admin permissions
                shutil.copyfile(str(fixture), str(dest))

Comment (Caligatio) [reply tests/conftest.py]: So os.symlink isn't available on Windows in Python 2 which would then maybe throw an `AttributeError` instead? I was thinking along the same lines as your try/except and also was unsure given the previous use of flags :)

Comment (brycedrennan) [reply tests/conftest.py]: Makes sense. Perhaps an inline comment is in order regarding the reasoning (admin privileges etc)?

Comment (Caligatio) [reply tests/conftest.py]: Added and pushed

**Pull request merged**